<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ECMC 물고기카운터</string>
  </property>
  <widget class="QLabel" name="common_text">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>25</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>커먼 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="uncommon_text">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>65</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>언커먼 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="rare_text">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>105</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>레어 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="epic_text">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>145</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>에픽 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="legendary_text">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>185</y>
     <width>87</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>레전더리 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="myth_text">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>225</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>신화 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="common_ea">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>25</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>마리</string>
   </property>
  </widget>
  <widget class="QLabel" name="uncommon_ea">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>66</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>마리</string>
   </property>
  </widget>
  <widget class="QLabel" name="rare_ea">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>106</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>마리</string>
   </property>
  </widget>
  <widget class="QLabel" name="epic_ea">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>145</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>마리</string>
   </property>
  </widget>
  <widget class="QLabel" name="legendary_ea">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>185</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>마리</string>
   </property>
  </widget>
  <widget class="QLabel" name="myth_ea">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>225</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>마리</string>
   </property>
  </widget>
  <widget class="QLabel" name="sum_text">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>307</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>합 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="sum_ea">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>307</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>마리</string>
   </property>
  </widget>
  <widget class="QLabel" name="sum">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>307</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="common">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>25</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="myth">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>225</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="uncommon">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>65</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rare">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>105</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="epic">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>145</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="legendary">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>185</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="percent0">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>25</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="percent5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>225</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="percent1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>65</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="percent2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>105</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="percent3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>145</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="percent4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>185</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="legendary_percent">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>185</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="epic_percent">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>145</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="common_percent">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>25</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="rare_percent">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>105</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="myth_percent">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>225</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="uncommon_percent">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>65</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pause_btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>카운트 중단</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>초기화</string>
   </property>
  </widget>
  <widget class="QLabel" name="version">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>333</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ver 1.2</string>
   </property>
  </widget>
  <widget class="QLabel" name="crab">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="crab_text">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>꽃게 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="crab_ea">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>마리</string>
   </property>
  </widget>
  <widget class="QLabel" name="percent6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="crab_percent">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="money_text">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>벌어들인 돈</string>
   </property>
  </widget>
  <widget class="QLabel" name="entropy_text">
   <property name="geometry">
    <rect>
     <x>374</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>엔트로피(물고기 손질)</string>
   </property>
  </widget>
  <widget class="QLabel" name="gold_ea">
   <property name="geometry">
    <rect>
     <x>488</x>
     <y>70</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>원</string>
   </property>
  </widget>
  <widget class="QLabel" name="entropy_ea">
   <property name="geometry">
    <rect>
     <x>474</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>엔트로피</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gold">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="entropy_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔스퀘어</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qtresource.qrc"/>
 </resources>
 <connections/>
</ui>
